sql/2020/19_Compression/party_of_content_encoding.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+#party of content encoding
+SELECT
+  client,
+  CASE
+   WHEN req_host!= ' ' THEN 'first-party'
+   ELSE 'thrid-party' END
+  AS party,
+  resp_content_encoding,
+  COUNT(0) AS num_requests,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
+FROM
+  `httparchive.sample_data.requests`
```

Comment:
The final query shouldn't use sample data

Reply:
revised.